ert3/evaluator/_evaluator.py

```diff
@@ -1,24 +1,102 @@
 import asyncio
-import os
 import pathlib
 import pickle
 import shutil
 from collections import defaultdict
 from pathlib import Path
-from typing import Any, Dict, List, Tuple
+from typing import Coroutine, Dict, Any, Tuple, List
+
 
 import cloudpickle
-from pydantic import FilePath
+
 import ert3
-from ert3.config import EnsembleConfig, StagesConfig, Step
-from ert3.data import EnsembleRecord, MultiEnsembleRecord, Record, RecordTransmitter
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
 from ert_shared.ensemble_evaluator.entity.identifiers import EVTYPE_EE_TERMINATED
 from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
 from ert_shared.ensemble_evaluator.prefect_ensemble import PrefectEnsemble
+from ert3.config import (
+    EnsembleConfig,
+    StagesConfig,
+    TransportableCommand,
+    Unix,
+    Function,
+    Step,
+)
+from ert3.data import (
+    EnsembleRecord,
+    MultiEnsembleRecord,
+    RealisationsToRecordToTransmitter,
+    RecordTransmitter,
+    Record,
+    SharedDiskRecordTransmitter,
+)
 
 _EVTYPE_SNAPSHOT_STOPPED = "Stopped"
 _EVTYPE_SNAPSHOT_FAILED = "Failed"
+_MY_STORAGE_DIRECTORY = ".my_storage"
+
+CoroutineTransmitters = List[Coroutine[Any, Any, None]]
+DictStrAny = Dict[str, Any]
+TupleDictStrAny = Tuple[DictStrAny, ...]
+
+
+def _add_storage(path: Path) -> Path:
+    if path.name != _MY_STORAGE_DIRECTORY:
+        return path / _MY_STORAGE_DIRECTORY
+    return path
```

Comment:
Isn't the input to `_add_storage` always `evaluation_tmp_dir`? Why the branching? Maybe you could explain with a comment in the code?

Reply:
_add_storage was more of a scaffolding function, forgot to remove it afterward.